scripts/link-validator.conf

```diff
@@ -0,0 +1,46 @@
+# SPDX-License-Identifier: Apache-2.0
+
+// config for https://github.com/ennru/site-link-validator/
+site-link-validator {
+  root-dir = "docs/target/paradox/site/main/"
+  # relative to `root-dir`
+  start-file = "index.html"
+
+  # Resolves URLs with the given prefix as local files instead
+  link-mappings = [
+    {
+      prefix = "https://pekko.apache.org/docs/pekko-persistence-cassandra/1.0.0/"
```

Comment:
So I don't think you are meant to be using 1.0.0 as a prefix here, only current i.e. https://github.com/apache/incubator-pekko/blob/442f54f6fe1da6857b96927c8bc42920b02e3aa0/scripts/link-validator.conf#L8-L39 as an example

Reply:
We use the 1.0.0 trick in other builds - just not in incubator-pekko.